python/tests/transformers/keras_transformer_test.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import numpy as np
+import os
+
+from keras.models import Sequential
+from keras.layers import Dense, Activation
+from keras.initializers import glorot_uniform
+
+from sparkdl.transformers.keras_transformer import KerasTransformer
+from ..tests import SparkDLTestCase
+
+class KerasTransformerTest(SparkDLTestCase):
+
+    # TODO(sid): How to create static class variables in Python? Should I just put these in the
+    # constructor?
+    NUM_FEATURES = 10
+    RANDOM_SEED = 997
+
+    def _loadNumpyData(self, num_examples, num_features):
+        """
+        Construct and return a 2D numpy array of shape (num_examples, num_features) corresponding
+        to a dataset of one-dimensional examples.
+        """
+        np.random.seed(self.RANDOM_SEED)
+        return np.random.randn(num_examples, num_features)
+
+    def getInputDF(self, sqlContext, inputCol, idCol):
```

Comment:
this could also be "private"? or have we been making functions used by `test_` methods "public"? whatever is consistent with other test classes works for me.